build/Dependencies.System2.props

```diff
@@ -95,7 +95,7 @@
 
     <SystemTextEncodingCodePages2PackageVersion>4.5.0</SystemTextEncodingCodePages2PackageVersion>
     <SystemTextJson2PackageVersion>4.7.1</SystemTextJson2PackageVersion>
-    <SystemTextEncodingsWeb2PackageVersion>4.7.2</SystemTextEncodingsWeb2PackageVersion>
+    <SystemTextEncodingsWeb2PackageVersion>4.6.0</SystemTextEncodingsWeb2PackageVersion>
```

Comment:
Are we downgrading all of these on purpose?

Reply:
Abandoned this PR in favor of staged PRs starting with [0] main -> [1]=+ CI -> [2] =+ APIMGateway -> [3] =+ Upstream -> [4] =+tlsAsyncWrite()
But will keep this in mind and ask you about correctness of future PRs